File: 1817-calculate-money-in-leetcode-bank/1817-calculate-money-in-leetcode-bank.cpp
class Solution {
public:
    int totalMoney(int n) {
        int k = 0;
        int l = 0;
        int sum = 0;
        
        while (l <= n) {
            for (int i = 1; i <= 7; i++) {
                sum += (k + i);
                l++;
                if (l >= n) {
                    return sum;
                }
            }
            k++;
        }
        
        return sum;
    }
};

File: 1817-calculate-money-in-leetcode-bank/1817-calculate-money-in-leetcode-bank.py
class Solution:
    def totalMoney(self, n: int) -> int:
        r=0
        k=0
        c=0
        while(r<=n):
            for i in range(1,8):
                c+=(k+i)
                r+=1
                if(r>=n):
                    return c
                
            k+=1
        return c
        